index.html

```diff
@@ -14,252 +14,400 @@
+              alt="card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (3).png"
+              alt="card"
+              class="mobile-images"
+            />
+          </li>
+        </ul>
       </section>
-
+    </main>
+    <section class="container">
       <section class="about-section">
-        <h2 class="myself">
-          About<br />
-          Myself
-        </h2>
-        <p class="career">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <h4 class="connect">LET’S CONNECT</h4>
-
-        <ul class="social-icons">
-          <li><span class="iconify" data-icon="iconoir:github"></span></li>
-          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
-          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
-          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
-          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
-        </ul>
-
-        <button class="resume">Get my resume</button>
+        <div class="career-container">
+          <h2 class="myself">
+            About<br />
+            Myself
+          </h2>
+          <p class="career">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h4 class="connect">LET’S CONNECT</h4>
+          <div class="social-career">
+            <ul>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="iconoir:github"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="eva:linkedin-fill"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="fontisto:angelist"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="ei:sc-twitter"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="bx:bxl-medium"></span>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <button class="resume">Get my resume</button>
+        </div>
 
-        <ul class="language-list">
-          <li class="langguage">
+        <div class="language-container">
+          <div class="langguage">
```

Comment:
### OPTIONAL

- [ ] This section space has not like the original. You can fix it.
- [ ] Also after frameworks has horizontal line. You can fix it.
- [ ] You can fix typo on 'Languguages'

| Yours | Original |
| ------------- |:-------------:|
| ![image](https://user-images.githubusercontent.com/38635158/144634356-e48fefe9-ff33-4269-94ee-324cba6925d5.png)       | ![image](https://user-images.githubusercontent.com/38635158/144634341-d66d7380-32bc-4903-9ddd-23fd9922d744.png)      |